modules/ROOT/pages/message-flows.adoc

```diff
@@ -19,55 +19,71 @@ Send messages to trading partners. An outbound message flow receives application
 
 Create an inbound message flow to receive EDI transactions from your trading partner, transform, and then send the translated application message to your backend systems.
```

Comment:
I think I have found a better wording to use globally for this:
```suggestion
Create an inbound message flow to receive and translate application messages from your partner and forward them to your backend systems:
```
(follow with diagram; skip all the repetitive text that follows:)